LunrCore/Index.cs

```diff
@@ -116,6 +117,92 @@ public class Index
             return builder.Build();
         }
 
+        #region Multi
+
+        public static class Fr
```

Comment:
Let's restructure this so language-specific implementations are neatly grouped under a `/globalization/[culture]`, such as `/globalization/fr` folder/namespace.

Reply:
@bleroy do you want it lowercase, as stated here, or dotnetified like `Globalization/Fr`?

Reply:
@bleroy I restructured in my last commit, using new static classes for the helper methods. Let me know if this is closer to what you were thinking.